sealtk/gui/Player.cpp

```diff
@@ -492,12 +494,12 @@ void PlayerPrivate::updateDetectedObjectVertexBuffers()
         {maxX, maxY},
         {maxX, minY},
       };
+      auto const bufSize =
+        detectedObjectVertexData.size() * static_cast<int>(sizeof(VertexData));
```

Comment:
Hehe... I had to run the tests to convince myself that this wasn't over the line limit :)